openml/runs/run.py

```diff
@@ -68,6 +69,21 @@ def _repr_pretty_(self, pp, cycle):
 
     @classmethod
     def from_filesystem(cls, folder):
+        """
+        The inverse of the to_filesystem method. Initiates a run based
```

Comment:
I think that `initiate` is unfortunate wording here as it also means 'to start'. How about 'instantiates a run object'?